### ISSUE TITLE
Project unification

### ISSUE THREAD

**Issue by jackinloadup** [schwal10/lunatic-fringe#23]
I just got a new comment/issue on my original project.

It seems you are taking doing a great job taking charge of this project. Would you like me to somehow refer or redirect people from my project to yours? jackinloadup/lunatic-fringe I'll start following your project to 

If you are not interested I think I'll post that anyone who wants can gain contributor access to the original project via a valid PR + request.

**Comment by schwal10** [comment]
I apologize for the delay in my response, I just noticed that this issue was added.

🤔 Unless you have strong feelings one way or the other, I am ok with leaving your project as the main project for the game without redirecting people to my project, especially since I have not been able to work on this much in a long while. I am going to PR my changes here to your repo shortly, that way your repo will be up to date with what I have and I can have someone else look at what I have added/changed thus far. If I have more time and am able to start working on it again consistently again in the future then we can redirect people to my project if you think it makes sense
